zcash_client_sqlite/src/wallet/init/migrations/v_sapling_shard_unscanned_ranges.rs

```diff
@@ -32,18 +29,24 @@ impl<P> schemer::Migration for Migration<P> {
     }
 }
 
-impl<P: consensus::Parameters> RusqliteMigration for Migration<P> {
+impl RusqliteMigration for Migration {
     type Error = WalletMigrationError;
 
     fn up(&self, transaction: &rusqlite::Transaction) -> Result<(), Self::Error> {
+        transaction.execute_batch(
```

Comment:
This should probably be in a separate migration; there are nightly Android builds of the SDK that are depending upon the existing migration. Also, the account birthday change is really a semantically separate thing from adding the view.